railties/lib/rails/command/behavior.rb

```diff
@@ -0,0 +1,123 @@
+require "active_support"
+
+module Rails
+  module Command
+    module Behavior #:nodoc:
+      extend ActiveSupport::Concern
+
+      class_methods do
+        # Remove the color from output.
+        def no_color!
+          Thor::Base.shell = Thor::Shell::Basic
+        end
+
+        # Track all command subclasses.
+        def subclasses
+          @subclasses ||= []
+        end
+
+        protected
+
+          # This code is based directly on the Text gem implementation.
+          # Copyright (c) 2006-2013 Paul Battley, Michael Neumann, Tim Fletcher.
+          #
+          # Returns a value representing the "cost" of transforming str1 into str2.
+          def levenshtein_distance(str1, str2)
```

Comment:
Sounds like the former would totally back up the latter here, right? Or am I missing something?